lib/api/merge_requests.rb

```diff
@@ -75,6 +75,22 @@ def handle_merge_request_errors!(errors)
         present merge_request, with: Entities::MergeRequest
       end
 
+
+      # Show MR files
+      #
+      # Parameters:
+      #   id (required)               - The ID of a project
+      #   merge_request_id (required) - The ID of MR
+      #
+      # Example:
+      #   GET /projects/:id/merge_request/:merge_request_id/files
+      #
+      get ":id/merge_request/:merge_request_id/files" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.